byway/src/main/java/com/google/sps/servlets/RecsServlet.java

```diff
@@ -31,6 +33,17 @@
   private final Gson gson = new Gson();
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+  // TODO: Remove when merging PR
```

Comment:
Is this just for local testing? Why have it as a part of this pull request if you plan on removing it when merging?

Reply:
I didn't want to keep adding it back in when testing if it's not finalized yet, since sometimes I want to start with a clean entity. But I will remove it for the PR.